README.md

```diff
@@ -539,9 +539,32 @@ neutron.add_router_interface router.id, subnet.id
 
 ## Development
 
-After checking out the repo, make sure rake is installed by running `gem install rake`. Afterwards, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+```
```

Comment:
LGTM. Even a copy-paste text would be fine (see https://github.com/ansible/ansible/blob/devel/hacking/README.md) 
```
$ git clone
$ cd fog-openstack
$ gem install rake            # ensure rake is installed        
$ bin/setup                       # install project dependencies 
```

Reply:
Are you ok with how it is right now? Gladly make it more copy-paste-like if you think that's better.